src/media/css/containers/login.styl

```diff
@@ -0,0 +1,20 @@
+@import '../lib'
+
+
+.login-form
+  background $color--white
+  border-radius 5px
+  max-width 400px
```

Comment:
It's `max-width`, we're good:

<img width="349" alt="screen shot 2015-09-29 at 1 39 15 pm" src="https://cloud.githubusercontent.com/assets/23885/10176021/821b4ac0-66af-11e5-81c2-3c03f19f081d.png">